scripts/downstream-runtime-env-setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#
+# This script setups the runtime environmental variables necessary to
+# run downstream type inference.
+#
+# Should be imported first by downstream inference starter script.
+# The following environmental variable should be specified in
+# downstream starter:
```

Comment:
`downstream starter` sounds weird...
What about `downstream projects' configuration scripts`?

Reply:
I'm referring to the `run-XXX.sh` script here. What name do you think is good for this?